src/index.js

```diff
@@ -2,15 +2,15 @@ import { TYPES, PixiComponent } from './utils/element'
 import { render } from './render'
 import Stage from './stage'
 import { PixiFiber } from './reconciler'
-import { AppProvider, AppConsumer, withPixiApp } from './stage/provider'
+import { AppProvider, AppConsumer, withPixiApp, Context } from './stage/provider'
 
 /**
  * -------------------------------------------
  * Public API
  * -------------------------------------------
  */
 
-export { render, Stage, AppProvider, AppConsumer, withPixiApp, PixiComponent, PixiFiber }
+export { render, Stage, AppProvider, AppConsumer, withPixiApp, PixiComponent, PixiFiber, Context }
```

Comment:
Let's rename it to `AppContext`

Reply:
Fixed!

Reply:
Awesome! Can we add a simple test for this?

Reply:
I will add it when I have time :)

Reply:
Awesome! I'll merge it once it's ready